app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,10 @@
 
     <%= stylesheet_link_tag "sticky-footer-navbar" %>
 
+    <style>
+      a {text-decoration: none;}
+    </style>
```

Comment:
Out of curiosity:

what's this?

Reply:
Yes it looks strange, but it is just to remove underline on devise links (user/shared/_links)

Reply:
I change it now, I set class `nav-link` on the links.